README.md

```diff
@@ -65,3 +65,7 @@ The following libraries / open-source projects were used in the development of d
  * [Sass](http://sass-lang.com/)
  * [Node.js](http://nodejs.org/)
  * [QUnit](https://github.com/jquery/qunit) / [jquery](http://jquery.com/)
+
+## Have a job for me?
+Please read [About Me](https://anhr.github.io/AboutMe/).
```

Comment:
Did you mean to add this?

Reply:
Currently I am looking for a job. Do you want to remove it from README.md? Do you have a job for me?

Reply:
Unfortunately this project is not the right place to put an available-for-hire post, sorry!

Reply:
ok